badge_server/test_badges.py

```diff
@@ -266,14 +266,47 @@
     GOOGLE_API_PYTHON_CLIENT_TENSORFLOW_RECENT_SUCCESS_3,
 ]
 
+UP_TO_DATE_DEPS = {
+    'google-auth': {
+        'current_time': datetime.datetime.utcnow(),
+        'installed_version': '1.6.3',
+        'installed_version_time': datetime.datetime(
+            2019, 2, 19, 21, 15, 56),
+        'is_latest': True,
+        'latest_version': '1.6.3',
+        'latest_version_time': datetime.datetime(
+            2019, 2, 19, 21, 15, 56)
+    },
+    'grpcio': {
+        'current_time': datetime.datetime.utcnow(),
+        'installed_version': '1.19.0',
+        'installed_version_time': datetime.datetime(
+            2019, 2, 27, 0, 0, 53),
+        'is_latest': True,
+        'latest_version': '1.19.0',
+        'latest_version_time': datetime.datetime(
+            2019, 2, 27, 0, 0, 53)
+    },
+    'google-api-core': {
```

Comment:
Don't use a dependency that we also test.
```suggestion
    'requests': {
```

Reply:
Without the 'google-api-core' dependency info, an error is triggered: https://github.com/GoogleCloudPlatform/cloud-opensource-python/blob/70b41cadaf7532cf03cee2e0bcc16f2706c0c5b7/compatibility_lib/compatibility_lib/compatibility_store.py#L194

(_This_ is case that I got mixed up with earlier today).

Reply:
`ValueError: missing version information for google-api-core`
Is the complete message

Reply:
In lieu of your discovery that a package must list itself as a dependency, this makes sense since the package being looked up is `google-api-core`.

Reply:
OK, I'd keep this dependency (and also add `requests`) but add a TODO saying that `google-api-core` should be removed when dependencies are properly added to the existing compatibility results

Reply:
My understanding was that `requests` was an arbitrary replacement for `google-api-core`. But given that I already have dummy data for other packages (`google-auth`, `grpcio`) why is it still necessary to add dummy data for `requests` in particular?

Reply:
Done